server/email/email-controller.js

```diff
@@ -0,0 +1,55 @@
+var mandrill = require('./email-service');
+var _ = require('underscore');
+
+module.exports = function(app, io){
+  var models = app.get('models');
+  var Gigs = models.Gig;
+  var UserGigs = models.UserGigs;
+  var Users = models.User;
+  var Locations = models.Location;
+
+  var emailController = {
+
+    sendEmails: function (req, res) {
+      var gigId = req.body.gigId;
+
+      Gigs.getGigInfo(gigId).then(function (gigInfo) {
```

Comment:
Agreed, would rather optimize for our code not erroring out over a efficiency increase unless we find that to be a bottleneck